CorePayments/src/test/java/com/paypal/android/corepayments/api/PatchCCOWithAppSwitchEligibilityUnitTest.kt

```diff
@@ -0,0 +1,302 @@
+package com.paypal.android.corepayments.api
+
+import android.content.Context
+import com.paypal.android.corepayments.CoreConfig
+import com.paypal.android.corepayments.Environment
+import com.paypal.android.corepayments.PayPalSDKError
+import com.paypal.android.corepayments.graphql.GraphQLClient
+import com.paypal.android.corepayments.graphql.GraphQLResult
+import com.paypal.android.corepayments.model.APIResult
+import com.paypal.android.corepayments.model.TokenType
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.mockk
+import io.mockk.slot
+import junit.framework.TestCase.assertEquals
+import junit.framework.TestCase.assertTrue
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import org.json.JSONObject
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@ExperimentalCoroutinesApi
+@RunWith(RobolectricTestRunner::class)
+class PatchCCOWithAppSwitchEligibilityUnitTest {
+
+    private lateinit var context: Context
+    private lateinit var coreConfig: CoreConfig
+    private lateinit var graphQLClient: GraphQLClient
+    private lateinit var authenticationSecureTokenServiceAPI: AuthenticationSecureTokenServiceAPI
+    private lateinit var sut: PatchCCOWithAppSwitchEligibility
+
+    private val testToken = "test-token-123"
+    private val testOrderId = "test-order-id-456"
+
+    @Before
+    fun beforeEach() {
+        context = mockk(relaxed = true)
+        coreConfig = CoreConfig("test-client-id", Environment.SANDBOX)
+        graphQLClient = mockk(relaxed = true)
+        authenticationSecureTokenServiceAPI = mockk(relaxed = true)
+        sut = PatchCCOWithAppSwitchEligibility(
+            authenticationSecureTokenServiceAPI,
+            graphQLClient
+        )
+    }
+
+    @Test
+    fun `invoke sends GraphQL request with correct authorization header`() = runTest {
+        // Given
+        val successResponse = createSuccessfulGraphQLResult()
+        coEvery { authenticationSecureTokenServiceAPI.createLowScopedAccessToken() } returns APIResult.Success(
+            testToken
+        )
+        coEvery { graphQLClient.send(any(), any(), any()) } returns successResponse
+
+        val headersSlot = slot<Map<String, String>>()
+
+        // When
+        sut.invoke(
+            context = context,
+            orderId = testOrderId,
+            tokenType = TokenType.ORDER_ID,
+            merchantOptInForAppSwitch = true,
+            paypalNativeAppInstalled = true
+        )
```

Comment:
```suggestion
        sut(
            context = context,
            orderId = testOrderId,
            tokenType = TokenType.ORDER_ID,
            merchantOptInForAppSwitch = true,
            paypalNativeAppInstalled = true
        )
```
🧶 : We shouldn't need to explicitly call `invoke` here, we can just use the `()` invocation operator.